src/coreclr/src/vm/rejit.cpp

```diff
@@ -1282,8 +1282,8 @@ HRESULT ReJitManager::GetReJITIDs(PTR_MethodDesc pMD, ULONG cReJitIds, ULONG * p
         CAN_TAKE_LOCK;
         GC_NOTRIGGER;
         PRECONDITION(CheckPointer(pMD));
-        PRECONDITION(pcReJitIds != NULL);
-        PRECONDITION(reJitIds != NULL);
+        PRECONDITION(pcReJitIds != NULL || cReJitIds == 0);
+        PRECONDITION((cReJitIds == 0) == (reJitIds == NULL));
```

Comment:
```suggestion
        PRECONDITION(pcReJitIds != NULL);
        PRECONDITION((cReJitIds == 0) || (reJitIds != NULL));
```

Reply:
For the same reason as above I don't want an || between the checks here, but I just pushed a commit simplifying the first condition to just `pcReJitIds != NULL`